python/veles/async_client/proto.py

```diff
@@ -35,9 +36,21 @@ def __init__(self):
         self.bids = {}
         self.rids = {}
         self.handlers = {}
+        self.auth_key = prepare_auth_key(key)
+
+    def _authorize(self):
+        self.transport.write(self.auth_key)
+        self.send_msg(messages.MsgConnect(
+            proto_version=1,
```

Comment:
I've moved the constant from ServerProto to module level and used it everywhere (I don't think updating server code and not updating client will be a common use-case).